src/modules/auth/resources/ResourcesListContainer.js

```diff
@@ -65,21 +89,53 @@ function ResourcesListContainer(props) {
 		} catch(e) {
 			console.error(e);
 			setLoading(false);
-			props.app.addAlert("warning", `${t("ResourcesListContainer|Something went wrong, failed to load resources")}. ${e?.response?.data?.message}`, 30);
+			props.app.addAlert("warning", `${t("ResourcesListContainer|Failed to load resources")}. ${e?.response?.data?.message}`, 30);
+		}
+	}
+	// Set terminate resource dialog
+	const terminateResourceForm = (resourceId) => {
+		var r = confirm(t('ResourcesListContainer|Do you want to delete this resource'));
+		if (r == true) {
+			deleteResource(resourceId);
 		}
 	}
 
-	const createResourceComponent = (
-		<ButtonWithAuthz
-			title={t("ResourcesListContainer|Create resource")}
-			color="primary"
-			onClick={() => {redirectToCreate()}}
-			resource="authz:superuser"
-			resources={credentialsResources}
-		>
-			{t("ResourcesListContainer|Create resource")}
-		</ButtonWithAuthz>
-	)
+	// Terminate the resource
+	const deleteResource = async (resourceId) => {
+		try {
+			let response = await SeaCatAuthAPI.delete(`/resource/${resourceId}`);
+			if (response.data.result !== "OK") {
+				throw new Error(t("ResourcesListContainer|Failed to delete the resource"));
+			}
+			props.app.addAlert("success", t("ResourcesListContainer|Resource successfully deleted"));
+			props.history.push("/auth/resources-deleted");
+		} catch(e) {
+			console.error(e);
+			props.app.addAlert("warning", `${t("ResourcesListContainer|Failed to delete the resource")}. ${e?.response?.data?.message}`, 30);
+		}
+	}
+
+	const customComponent = (
+		<div style={{display: "flex"}} >
+			<Button
```

Comment:
this should be only for superusers i think. also, as it is implemented now on backend, it should probably be a switch or checkbox "Include deleted resources". because this button "Deleted resources" looks like it will show you a list of deleted resources which is not true.

Reply:
@petrKavulok So then in should be `<ButtonWithAuthz />` with appropriate resource